Lib/test/_test_multiprocessing.py

```diff
@@ -775,12 +775,20 @@ def test_sys_exit(self):
 
             os.unlink(testfn)
 
-        for reason in (True, False, 8):
-            p = self.Process(target=sys.exit, args=(reason,))
+        cases = [
+            ((True,), 1),
+            ((False,), 0),
+            ((8,), 8),
+            ((None,), 0),
+            ((), 0),
+            ]
+
+        for args, expected in cases:
```

Comment:
can probably write these as [subtests](https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests)

also probably add a case for the non-int, non-None case

Reply:
Updated to use `self.subTest`. The non-int, non-None cases are covered by the set of tests above (`[1, 2, 3]`, `'ignore this'`).